lib/wallet/wallet.js

```diff
@@ -1736,6 +1741,52 @@ class Wallet extends EventEmitter {
     return mtx.signAsync(rings, Script.hashType.ALL, this.wdb.workers);
   }
 
+  /**
+   * Get all pending ancestors.
+   * @param {TX} tx
+   * @returns {Promise} - Returns {@link Hash}[]
+   */
+
+   async getPendingAncestors(tx) {
+    return this._getPendingAncestors(tx, new BufferSet());
+   }
+
+  /**
+   * Get all pending ancestors.
+   * @param {TX} tx
+   * @param {Object} set
+   * @returns {Promise} - Returns {@link Hash}[]
```

Comment:
Type of the `set` is `{BufferSet<Hash>}` and returns `{Promise<BufferSet<Hash>>}`

Reply:
I haven't seen that method of jsdocs anywhere in bcoin.

Reply:
That may be something for another pull request.

Reply:
That's true. But in this case it should be something like:
```
  @param {BufferSet} set - {@link Hash}
  @returns {Promise} - {@link BufferSet} ...
```

Reply:
`@param {Object} set` is used throughout mempool, but could be a target for a future JSdocs PR (I have a long list of nits for that future PR!)

I'm not sure any we return a BufferSet anywhere else, do we?

Reply:
Yeah, agree. We need JSDoc updates all over the place. Here I was just mentioning that we don't return {Hash}[]. Not that we should switch to this syntax.

Reply:
switched to `{@link MempoolEntry[]}` thanks - that one has changed a few times in this review :-)